src/scss/settings/_colors.scss

```diff
@@ -3,9 +3,7 @@
   * @stylesheet gaiden_css.settings.colors.css Colors
   *
   * There are different colors that can be used:
-  * @iframe ../../demo/variables/colors/demo.html
-  *
-  *
+  * @iframe ../../demo/variables/colors/demo.html 360
```

Comment:
@eduardojmatos o.O oh gosh, i check it when before push .... my bad. I'll fix this.